### PR TITLE
Update Badge component

### PR DESCRIPTION
**Pull request by anisha-deverashetty**
Updated Badge component to have more variations. Updated the design for Generic type

**Badge without icon**
image
**Badge with max char count**
image
**Generic**
image

### CODE REVIEW

File: explorer/src/Stories/Badge.elm [hunk 1]
```diff
@@ -45,4 +45,28 @@ stories =
...
+                Html.div [ css [ displayFlex ] ]
+                    [ Badge.Dot |> Badge.view [] [ Icons.calendar [ css [ width (rem 2) |> Css.important ] ] ] ]
+          , {}
+          )
+        , ( "Dot no icon"
+          , \_ ->
+                Html.div [ css [ displayFlex ] ]
+                    [ Badge.Dot |> Badge.view [] [] ]
+          , {}
+          )
```
Comment (fredrikloberg) [inline explorer/src/Stories/Badge.elm:71]: Do we need this dot variant? Best if we could keep it simple and only have one generic variant

Comment (anisha-deverashetty) [reply explorer/src/Stories/Badge.elm:71]: We have the design here. The dot is the one we need to use when we don't have the number. I couldn't find the generic variant in design lib and Line too didn't know about this one. 
image

Comment (anisha-deverashetty) [reply explorer/src/Stories/Badge.elm:71]: If we are not using this generic one then it should be replaced to dot. What you think?

Comment (fredrikloberg) [reply explorer/src/Stories/Badge.elm:71]: Yes I agree, challenge Line about replacing the old generic one rather than adding a new

**Pull request merged**